<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkflowSettingsWidget</class>
 <widget class="QWidget" name="WorkflowSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Scene appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="gridBox">
        <property name="text">
         <string>Show grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="snapBox">
        <property name="text">
         <string>Snap to grid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="styleCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Element style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fontCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Element font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="colorWidget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Element background color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Runtime settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="lockBox">
        <property name="text">
         <string>Track running progress</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Directories</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Directory for custom elements with scripts</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="dirEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="dirButton">
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Directory for custom elements with command line tools</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="extToolDirEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="extToolDirButton">
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="runInSeparateProcessBox">
     <property name="text">
      <string>Run tasks in separate process</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>178</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
